modules/shared_account_child_access/aws_nuke.tf

```diff
@@ -7,8 +7,56 @@ resource "aws_iam_role" "assume_aws_nuke_access" {
   tags = var.tags
 }
 
-resource "aws_iam_role_policy_attachment" "assume_nuke_access_admin_access" {
+resource "aws_iam_role_policy" "nuke_policy_1" {
+  count  = (var.create_nuke_user == true) ? 1 : 0
+  name   = "NukePolicy1"
+  role   = aws_iam_role.assume_aws_nuke_access[count.index].id
+  policy = file("${path.module}/policies/allow_nuke_part1.json")
+}
+
+# tfsec:ignore:aws-iam-no-policy-wildcards
+resource "aws_iam_role_policy" "nuke_policy_2" {
+  count  = (var.create_nuke_user == true) ? 1 : 0
+  name   = "NukePolicy2"
+  role   = aws_iam_role.assume_aws_nuke_access[count.index].id
+  policy = file("${path.module}/policies/allow_nuke_part2.json")
+}
+
+resource "aws_iam_role_policy" "nuke_policy_3" {
+  count  = (var.create_nuke_user == true) ? 1 : 0
+  name   = "NukePolicy3"
+  role   = aws_iam_role.assume_aws_nuke_access[count.index].id
+  policy = file("${path.module}/policies/allow_nuke_part3.json")
+}
+
+resource "aws_iam_role_policy" "nuke_policy_4" {
+  count  = (var.create_nuke_user == true) ? 1 : 0
+  name   = "NukePolicy4"
+  role   = aws_iam_role.assume_aws_nuke_access[count.index].id
+  policy = file("${path.module}/policies/allow_nuke_part4.json")
+}
+
+resource "aws_iam_role_policy_attachment" "attach_nuke_access_1" {
   count      = (var.create_nuke_user == true) ? 1 : 0
-  policy_arn = "arn:aws:iam::aws:policy/AdministratorAccess"
+  policy_arn = aws_iam_role_policy.nuke_policy_1[count.index].arn
```

Comment:
You don't need to attach these like so, a `aws_iam_role_policy_attachment` is to attach a role to a policy, an `aws_iam_role_policy` will create the policy inline in the role, if you want to do it as an attachment, then create the first resource as a `aws_iam_role`